ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ /*global require*/
+ var $ = require('jquery');
+ var Backbone = require('backbone');
+ var user = require('component/singletons/user-instance');
+
+ let resultTemplates = [];
+ let resultTemplatePromise = $.ajax({
+    type: 'GET',
+    context: this,
+    url: '/search/catalog/internal/forms/result',
+    contentType: 'application/json',
+    success: function (data) {
+        resultTemplates = data;
+    }
+ });
+
+ module.exports = Backbone.Model.extend({
+    initialize: function (options) {
+        this.options = options;
+        resultTemplatePromise.then(() => {
+            if (!this.isDestroyed) {
+                var customResultTemplates = _.map(resultTemplates, function(resultForm) {
```

Comment:
Since this is a new file, `const` would be preferable.

Reply:
done